packages/meteor-gazelle-accounts/package.js

```diff
@@ -11,7 +11,9 @@ Package.onUse(function (api) {
 
   var packages = [
     'meteor-gazelle:lib',
-    'useraccounts:unstyled'
+    'useraccounts:unstyled',
+    'meteor-gazelle:ipmanager',
```

Comment:
meteor-gazelle:ip-manager

How to handle packages that have spaces in the name?